server/rpc_test.go

```diff
@@ -338,6 +338,14 @@ func TestGenPayment(t *testing.T) {
 	sender := &pm.MockSender{}
 	s.Sender = sender
 
+	// Test invalid price
+	BroadcastCfg.SetMaxPrice(big.NewRat(1, 5))
+	payment, err = genPayment(s, 1)
+	assert.Equal("", payment)
+	assert.Error(err)
```

Comment:
Check that this is the error that we expect?

Reply:
fixed in https://github.com/livepeer/go-livepeer/pull/992/commits/18648d78c0d532e7e1cc26d8bc1e374a560cee34